netlify.toml

```diff
@@ -4,17 +4,24 @@
 
 # production
 [context.master]
-  environment = { HOST="https://applicant-portal.herokuapp.com/" }
+  environment = {
```

Comment:
From the [TOML language docs](https://github.com/toml-lang/toml#inline-table):
> it is strongly discouraged to break an inline table onto multiples lines. If you find yourself gripped with this desire, it means you should be using standard tables.

If we want to split this on multiple lines, we should do it like this, which probably makes more sense as we may end up having to add more environment variables in the future:
```toml
[context.master.environment]
  HOST="..."
  NYCID_CLIENT_ID="..."
```
(oh I see you brought them on the same line.  I think the curly brackets need to be on the same line, too)